internal/auth/handler.go

```diff
@@ -34,4 +34,11 @@ type Handler interface {
 		RemoteAddress string,
 		ConnectionID string,
 	) (bool, map[string]string, error)
+
+	OnAuthorization(
```

Comment:
Please add a comment to this function and explain how to use this. I would also consider renaming it to make it explicit that this is for Kerberos.

Reply:
This is deliberate. I was thinking other auth backends (i.e. GitHub ?) can use this for authz - this also solves the `EnforceUsername` problems of the GitHub backend if this is added to that as well.

Reply:
Works for me. Let's document accordingly.